### DISCUSSION TITLE
Add new Ysort node that can work with any nodes, not only children

### DISCUSSION THREAD

**Discussion by me2beats** [godotengine/godot-proposals#4600] [Engine Core]
I had cases when I need to use Ysort, that could work with nodes which are not Ysort children.
So I had to remove those nodes and add them to the Ysort, which is not so handy.

I think a new Ysort node could be added that will use nodes paths for example rather than children

**Comment by Zireael07** [comment]
I believe we already have an open proposal for something like that? https://github.com/godotengine/godot-proposals/issues/1237

**Reply by me2beats** [reply]
`deep` means it still uses children (and grandchildren etc).
but what if the node isn't there?
you still would remove_child()/add_child()
